Cubical/Data/Int/Properties.agda

```diff
@@ -1,4 +1,28 @@
 {-# OPTIONS --cubical --safe #-}
+
+{-
+
+This file defines
```

Comment:
I leave `{-# OPTIONS --cubical --safe #-}` in line one. Let me know if you prefer it come after the header description. For me OPTIONS in line 1 seems cleaner. (Do we have a particular style yet?)

Reply:
No we don't, see #19 (it's about naming but I believe we don't have other kind of convensions right now)

Reply:
I don't have any preferences. Any of them are easy to read